pkg/utils.go

```diff
@@ -56,18 +58,23 @@ type mariadbOptions struct {
 	dumpOptions   restic.DumpOptions
 }
 
-func waitForDBReady(appBinding *v1alpha1.AppBinding, secret *core.Secret, waitTimeout int32) error {
+func (opt *mariadbOptions) waitForDBReady(appBinding *v1alpha1.AppBinding, secret *core.Secret) error {
 	log.Infoln("Waiting for the database to be ready.....")
 	shell := sh.NewSession()
 	shell.SetEnv(EnvMariaDBPassword, string(secret.Data[MariaDBPassword]))
 	args := []interface{}{
 		"ping",
 		"--host", appBinding.Spec.ClientConfig.Service.Name,
-		"--user=root",
-		fmt.Sprintf("--wait=%d", waitTimeout),
+		"--user", string(secret.Data[MariaDBUser]),
 	}
+	spew.Dump(args)
```

Comment:
Do you need this anymore?

Reply:
No. I used that for testing. Missed to remove that.